scene/3d/xr_body_modifier_3d.cpp

```diff
@@ -44,13 +44,9 @@ void XRBodyModifier3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_bone_update", "bone_update"), &XRBodyModifier3D::set_bone_update);
 	ClassDB::bind_method(D_METHOD("get_bone_update"), &XRBodyModifier3D::get_bone_update);
 
-	ClassDB::bind_method(D_METHOD("set_show_when_tracked", "show"), &XRBodyModifier3D::set_show_when_tracked);
-	ClassDB::bind_method(D_METHOD("get_show_when_tracked"), &XRBodyModifier3D::get_show_when_tracked);
-
 	ADD_PROPERTY(PropertyInfo(Variant::STRING, "body_tracker", PROPERTY_HINT_ENUM_SUGGESTION, "/user/body"), "set_body_tracker", "get_body_tracker");
```

Comment:
This suggestion still needs an update:

```suggestion
	ADD_PROPERTY(PropertyInfo(Variant::STRING, "body_tracker", PROPERTY_HINT_ENUM_SUGGESTION, "/user/body_tracker"), "set_body_tracker", "get_body_tracker");
```

And so does the one in `xr_face_modifier_3d.cpp`, but GitHub won't let me put an inline comment on it because it's not part of the diff - see: https://github.com/godotengine/godot/pull/90645/files#diff-b57ec6aef52bc7c767c24a628dce99ecdfbb018f0d083d333aff94ddedd7a612R498

That one should be like:

```diff
diff --git a/scene/3d/xr_face_modifier_3d.cpp b/scene/3d/xr_face_modifier_3d.cpp
index 655ef3d26c..43cef95fb9 100644
--- a/scene/3d/xr_face_modifier_3d.cpp
+++ b/scene/3d/xr_face_modifier_3d.cpp
@@ -495,7 +495,7 @@ static void remove_driven_unified_blend_shapes(RBMap<int, int> &p_blend_mapping)
 void XRFaceModifier3D::_bind_methods() {
        ClassDB::bind_method(D_METHOD("set_face_tracker", "tracker_name"), &XRFaceModifier3D::set_face_tracker);
        ClassDB::bind_method(D_METHOD("get_face_tracker"), &XRFaceModifier3D::get_face_tracker);
-       ADD_PROPERTY(PropertyInfo(Variant::STRING, "face_tracker", PROPERTY_HINT_ENUM_SUGGESTION, "/user/head"), "set_face_tracker", "get_face_tracker");
+       ADD_PROPERTY(PropertyInfo(Variant::STRING, "face_tracker", PROPERTY_HINT_ENUM_SUGGESTION, "/user/face_tracker"), "set_face_tracker", "get_face_tracker");
 
        ClassDB::bind_method(D_METHOD("set_target", "target"), &XRFaceModifier3D::set_target);
        ClassDB::bind_method(D_METHOD("get_target"), &XRFaceModifier3D::get_target);
```

Reply:
Whups - I believe I caught all of these now (global search for "/user/" in the repo is very noisy). I also found I had forgotten to implement the "show_when_tracked" logic.